external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaSinkSuite.scala

```diff
@@ -515,3 +591,14 @@ class KafkaSinkBatchSuiteV2 extends KafkaSinkBatchSuiteBase {
     }
   }
 }
+
+class TestKafkaPartitioner extends DefaultPartitioner {
+  override def partition(
+      topic: String,
+      key: Any,
+      keyBytes: Array[Byte],
+      value: Any,
+      valueBytes: Array[Byte],
+      cluster: Cluster)
+  : Int = 0
```

Comment:
nit: append this to the previous line

Reply:
According to the style guide, https://github.com/databricks/scala-style-guide#spacing-and-indentation , the return type on a new line is also acceptable (and I like it better :)

Reply:
you can take a look at other codes and see how the code looks like. most likely we don't prefer unnecessary line.

Reply:
+1 sticking to the available code style.

Reply:
Alright then. Changed.